addon-test-support/@ember/test-helpers/setup-context.js

```diff
@@ -14,18 +14,75 @@ import { nextTickPromise } from './-utils';
 
 let __test_context__;
 
+/**
+  Stores the provided context as the "global testing context".
+
+  Generally setup automatically by `setupContext`.
+
+  @public
+  @method setContext
+  @param {Object} context the context to use
+*/
 export function setContext(context) {
   __test_context__ = context;
 }
 
+/**
+  Retrive the "global testing context" as stored by `setContext`.
+
+  @public
+  @method getContext
+  @returns {Object} the previously stored testing context
+*/
 export function getContext() {
   return __test_context__;
 }
 
+/**
+  Clear the "global testing context".
+
+  Generally invoked from `teardownContext`.
+
+  @public
+  @method unsetContext
+*/
 export function unsetContext() {
   __test_context__ = undefined;
 }
 
+/**
+  Returns a promise to be used to pauses the current test (due to being
+  returned from the test itself).  This is useful for debugging while testing
+  or for test-driving.  It allows you to inspect the state of your application
+  at any point.
+
+  @example <caption>Usage via ember-qunit</caption>
+
+  ```js
+  import { setupRenderingTest } from 'ember-qunit';
+  import { render, click, pauseTest } from '@ember/test-helpers';
+
+  module('awesome-sauce', function(hooks) {
+    setupRenderingTest(hooks);
+
+    test('does something awesome', function(assert) {
+      render(hbs`{{awesome-sauce}}`);
```

Comment:
this example is not using async/await, which seems a little strange as it's the default now 🤔

Reply:
Totally agreed. Will update.